Userland/Libraries/LibGfx/Color.cpp

```diff
@@ -83,178 +83,171 @@ Optional<Color> Color::from_string(StringView string)
         return {};
 
     struct ColorAndWebName {
-        constexpr ColorAndWebName(ARGB32 c, char const* n)
-            : color(c)
-            , name(n != nullptr ? StringView { n, __builtin_strlen(n) } : StringView {})
-        {
-        }
         ARGB32 color;
         StringView name;
     };
 
     constexpr ColorAndWebName web_colors[] = {
```

Comment:
make that an `AK::Array` while we're here, pls :)

Reply:
Then we have to say `ColorAndWebName` for each entry or it won't infer the length (which is super annoying)

Reply:
I guess it's not too bad with a nicer name :)

Reply:
Yeah, seems ok. I'm sure some template magician can figure out how to make it infer the type and length eventually!